apps/etterlatte-brev-api/src/main/kotlin/no/nav/etterlatte/brev/VedtaksbrevRoute.kt

```diff
@@ -0,0 +1,29 @@
+package no.nav.etterlatte.brev
+
+import io.ktor.server.application.call
+import io.ktor.server.request.receive
+import io.ktor.server.response.respond
+import io.ktor.server.routing.Route
+import io.ktor.server.routing.post
+import io.ktor.server.routing.route
+import org.slf4j.LoggerFactory
+
+fun Route.vedtaksbrevRoute(service: VedtaksbrevService) {
+    val logger = LoggerFactory.getLogger("no.nav.etterlatte.brev.VedaksbrevRoute")
```

Comment:
Hvorfor ikke bare: `val logger = LoggerFactory.getLogger(this::class.java)` ?

Reply:
Tenker det er potejto potato, litt avhengig av smak og behag. Ved flere refaktoreringer trenger man ikke være obs på pakkenavn dersom this::class.java brukes; det må man ved å bruke en absolutt sti.